scripts/base/frameworks/notice/main.zeek

```diff
@@ -136,6 +136,9 @@ export {
 		## The actions which have been applied to this notice.
 		actions:        ActionSet      &log &default=ActionSet();
 
+		## The email address(es) where to send this notice
+		email_dest:     set[string]    &log &optional;
```

Comment:
Wondering if we could just make this non-optional and default to an empty set? Then the code accessing it would become easier as it didn't need to check for not-set each time.

Reply:
Yeah, I think that makes sense. I'll make that change and update the baselines.

Here's another question: Should `add` on an uninitialized set just initialize it first? I think having these as optional is a pretty common pattern, and it would clean up a lot of code.

Reply:
> Here's another question: Should `add` on an uninitialized set just initialize it first? I think having these as optional is a pretty common pattern, and it would clean up a lot of code.

Yeah, I can see that. It's a bit of magic in return for gaining some convenience, but seems it would quite intuitive. Not sure if it's easy to implement though because we'd need special-case `add` on record fields.